GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -690,13 +728,7 @@ private void AddRemoteClick(object sender, EventArgs e)
         {
             // store the selected remote name
             string selectedRemoteName = _selectedRemote.Name;
```

Comment:
Remove this useless variable (and comment), now

Reply:
umm what?

Reply:
Now that you replaced all the code by a method, this `selectedRemoteName` var is useless (and the comment, too). Its name don't bring more information.
Replace by ` ManageRemotes(_selectedRemote.Name);`